Bug fixes.

MazharulIslam-Naim opened the pull request:
[Fixed a bug with actions page not having anything in localStorage. M
oved challenger crown slightly over. Fixed a bug where signin would infinitely loop. Made sure take action shows up for the correct users. Fixed bug with invite preview not loading.

MazharulIslam-Naim committed:
Fixed a bug with actions page not haveing anything in localStorage. Moved challenger crown slightly over. Fixed a bug where signin would infintly loop. Made sure take action shows up for the correct users. Fixed bug with invite preview not loading.

MazharulIslam-Naim committed:
Merge branch 'development' into bugs

rickyyx commented:
@Devanshusp could you help review this one?

Devanshusp commented on src/components/Actions/Actions.js:
```diff
@@ -44,18 +44,19 @@ export default function Actions() {
               setNotifyElectionReminders(data.notifyElectionReminders);
               getChallengerInfo(data.invitedBy);
-              setLoading(false);
+              setLoading(true);
```
Remove `setLoading(true)` from here and add at the end of getChallengerInfo function

Need to wait for the getChallengerInfo function to execute before loading data when nothing is in localStorage, otherwise this issue comes up because we are trying to load data that hasn't been populated yet
![image](https://user-images.githubusercontent.com/66650721/171069492-cb2873c9-222d-4f17-8c8f-68f52696e4ec.png)

Devanshusp commented on src/components/Actions/Actions.js:
```diff
@@ -67,13 +68,15 @@ export default function Actions() {
     info.challengerID = invitedBy;
-    setChallengerInfo(JSON.stringify(info));
+    setChallengerInfo(info);
+    localStorage.setItem("challengerInfo", JSON.stringify(info));
```
Add `setLoading(true)` here instead

Devanshusp reviewed (changes requested):
Hey @MazharulIslam-Naim, thanks for working on this! Didn't know some of these bugs existed- nice catch!
I did find a minor bug with the loading in /actions page

Btw, could you point me to how you fixed the invite preview not loading?

MazharulIslam-Naim committed:
Moved loading end to after the challengers info is gotten. Removed success endpoint because mata tags didn't appear as intended.

MazharulIslam-Naim commented:
Hey @Devanshusp I fixed the the setLoading. The problem was because of playerwelcome used in preview and for the invite preview there would be nothing would be set to the variable challengerInfo and it would not get out of loading so I changed it to set the current user's info when playerwelcome is used on invite preview.(look at changes to playerwelcome component)

Devanshusp reviewed (approved):
Thanks for fixing the bugs!
